src/main/proto/control.proto

```diff
@@ -23,6 +23,8 @@ message PlatformOutboundInstruction {
         StartEventProcessor start_event_processor = 5;
         ReleaseEventProcessorSegment release_segment = 6;
         RequestEventProcessorInfo request_event_processor_info = 7;
+        SplitEventProcessorSegment split_event_processor_segment = 8;
```

Comment:
I would rather not have to change the API for this operation. If we can use the existing "serialized object" approach, that would be more powerful for the future.
I am also fine accepting this, for now, and make the API in 4.2 more generic, also allowing for results of the instructions to be passed back to AxonServer.

Reply:
The existing message we could leverage from a required parameters perspective for this is the `ReleaseEventProcessorSegment`, although I find the naming for that quite out of place to use it for split and merge too.
Additionally, I am not entirely sure what you're angling for in regards to 'the existing "serialized object" approach'.